multi_map_it_test.go

```diff
@@ -539,6 +539,26 @@ func TestMultiMap_ContainsEntry(t *testing.T) {
 	})
 }
 
+func TestMultiMap_ValueCount(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		key := "key"
+		targetValues := []interface{}{"v1", "v2", "v3"}
+		it.Must(m.PutAll(ctx, key, targetValues...))
+		count, err := m.ValueCount(ctx, key)
+		if err != nil {
+			t.Error(err)
+		}
+		assert.EqualValues(t, count, len(targetValues))
+		nonExistingKey := "dummyKey"
+		count, err = m.ValueCount(ctx, nonExistingKey)
```

Comment:
You can simplify this one to:
```
count, err = m.ValueCount(ctx, "non-existing-key")
```

Reply:
Fixed, [8d5d17e](https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/8d5d17e46c48d1f3b6dbecd61ba9881f11c6a06a).